server/src/test/java/org/elasticsearch/common/settings/ScopedSettingsTests.java

```diff
@@ -54,11 +54,46 @@
 import static org.hamcrest.CoreMatchers.instanceOf;
 import static org.hamcrest.CoreMatchers.startsWith;
 import static org.hamcrest.Matchers.arrayWithSize;
+import static org.hamcrest.Matchers.empty;
 import static org.hamcrest.Matchers.hasToString;
 import static org.hamcrest.Matchers.sameInstance;
 
 public class ScopedSettingsTests extends ESTestCase {
 
+    private static class FooLowSettingValidator implements Setting.Validator<Integer> {
```

Comment:
I think these settings can reasonably be moved right next to `testUpdateOfValidationDependantSettings` since that's the only place they're used.

Reply:
Yes, that would be better, I'll move them.